src/policies/alpha_vector.jl

```diff
@@ -0,0 +1,59 @@
+######################################################################
+# alpha_vector.jl
+#
+# implements policy that is a set of alpha vectors
+######################################################################
+
+struct AlphaVectorPolicy{P<:POMDP, A} <: Policy
+    pomdp::P
+    alphas::Vector{Vector{Float64}}
+    action_map::Vector{A}
+end
+function AlphaVectorPolicy(pomdp::POMDP, alphas)
+    AlphaVectorPolicy(pomdp, alphas, ordered_actions(pomdp))
+end
+# assumes alphas is |S| x |A|
+function AlphaVectorPolicy(p::POMDP, alphas::Matrix{Float64}, action_map)
+    # turn alphas into vector of vectors
+    num_actions = size(alphas, 2)
+    alpha_vecs = Vector{Float64}[]
+    for i = 1:num_actions
+        push!(alpha_vecs, vec(alphas[:,i]))
+    end
+
+    AlphaVectorPolicy(p, alpha_vecs, action_map)
+end
+
+
+
+updater(p::AlphaVectorPolicy) = DiscreteUpdater(p.pomdp)
+
+value(p::AlphaVectorPolicy, b::DiscreteBelief) = value(p, b.b)
+function value(p::AlphaVectorPolicy, b::Vector{Float64})
+    num_vectors = length(p.alphas)
+    max_value = -Inf
+    for i = 1:num_vectors
+        temp_value = dot(b, p.alphas[i])
+        temp_value > max_value && (max_value = temp_value)
```

Comment:
minor style point (optional to fix). This short circuiting trick is harder to read (especially for new programmers). I think a standard if statement is better.

Or you could just replace the whole thing with
```
value(p::AlphaVectorPolicy, b::Vector{Float64}) = maximum(dot(b, a) for a in p.alphas)
```